web/viewer.css

```diff
@@ -1117,6 +1117,7 @@ a:is(.toolbarButton, .secondaryToolbarButton)[href="#"] {
   /*#endif*/
 
   .loadingInput:has(> &.loading)::after {
+    display: block;
     visibility: visible;
 
     transition-property: visibility;
```

Comment:
Could we possibly use `display` instead here, and thus remove the `visibility`-rules from these elements?

Reply:
Unfortunately, `display` isn't animatable.
Just to be sure I tried and I confirm that it doesn't work.

Reply:
That's too bad; thanks for checking!